cms/static/js/spec/views/pages/course_outline_spec.js

```diff
@@ -872,50 +876,66 @@ define(["jquery", "common/js/spec_helpers/ajax_helpers", "common/js/components/u
                     expect($("#due_date").val()).toBe('7/10/2014');
                     expect($("#grading_type").val()).toBe('Lab');
                     expect($("#staff_lock").is(":checked")).toBe(true);
-                    expect($("#id_timed_exam").is(":checked")).toBe(false);
-                    expect($("#id_proctored_exam").is(":checked")).toBe(true);
-                    expect($("#id_not_timed").is(":checked")).toBe(false);
-                    expect($("#id_practice_exam").is(":checked")).toBe(false);
-                    expect($("#id_time_limit").val()).toBe("02:30");
+                    expect($("input.timed_exam").is(":checked")).toBe(true);
+                    expect($("input.proctored_exam").is(":checked")).toBe(false);
+                    expect($("input.no_special_exam").is(":checked")).toBe(false);
+                    expect($("input.practice_exam").is(":checked")).toBe(false);
+                    expect($(".field-time-limit input").val()).toBe("02:30");
+                    expect($(".field-hide-after-due input").is(":checked")).toBe(true);
                 });
 
-                it('can hide the time limit field when the None radio box is selected', function() {
+                it('can hide time limit and hide after due fields when the None radio box is selected', function() {
                     createCourseOutlinePage(this, mockCourseJSON, false);
                     outlinePage.$('.outline-subsection .configure-button').click();
                     setEditModalValues("7/9/2014", "7/10/2014", "Lab", true);
+                    selectAdvancedSettings();
                     selectDisableSpecialExams();
 
-                    // id_time_limit_div should be hidden when None is specified
-                    expect($('#id_time_limit_div')).toHaveClass('is-hidden');
+                    // all additional options should be hidden
+                    expect($('.exam-options').is(':hidden')).toBe(true);
                 });
 
                 it('can select the practice exam', function() {
                     createCourseOutlinePage(this, mockCourseJSON, false);
                     outlinePage.$('.outline-subsection .configure-button').click();
                     setEditModalValues("7/9/2014", "7/10/2014", "Lab", true);
+                    selectAdvancedSettings();
                     selectPracticeExam("00:30");
-                    // id_time_limit_div should not be hidden when practice exam is specified
-                    expect($('#id_time_limit_div')).not.toHaveClass('is-hidden"');
+
+                    // time limit should be visible, review rules and hide after due should be hidden
+                    expect($('.field-time-limit').is(':visible')).toBe(true);
```

Comment:
Nit: since these 3 things are often checked together, create a helper method for verifying their state.